core/utils.ts

```diff
@@ -135,6 +140,15 @@ export function validate(compiledGraph: dataform.ICompiledGraph): dataform.IGrap
         const message = `Invalid value for sqldatawarehouse distribution: "${distribution}"`;
         validationErrors.push(dataform.ValidationError.create({ message, actionName }));
       }
+
+      if (action.uniqueKeys) {
```

Comment:
This is inside the wrong IF statement, you don't care about if the distribution is set or not (line 132).

Reply:
Oops, thanks